src/interactive/infoToggle.story.jsx

```diff
@@ -2,11 +2,50 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { decorateWithBasics } from '../utils/decorators';
 import { withInfo } from '@storybook/addon-info';
-import { withKnobs, text, object, select } from '@storybook/addon-knobs';
+import { withKnobs, text, object, select, boolean } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 
 import InfoToggle from './InfoToggle';
 
+class ManualInfoToggle extends React.PureComponent {
+	state = {
+		tooltipOpen: false,
+	};
+
+	toggleTooltip = () => {
+		this.setState(prevState => ({ tooltipOpen: !prevState.tooltipOpen }));
```

Comment:
avoid calling the arg to the updater function `prevState` - it's actually the _future_ value of state when state updates are being applied, so `prevState` is misleading. Just call it `state`

Reply:
https://reactjs.org/docs/react-component.html#setstate 
according to the docs it is referencing the previous state

Reply:
ah, I stand corrected. I still find it odd since it could be a future value relative to `this.state`, but ok.

Reply:
maybe i should use this as an excuse to to tweet @ dan_abramov

Reply:
💯 i support it haha

Reply:
if he ever answers... https://twitter.com/mikemcgahan/status/1037074327269568512

Reply:
React docs update PR https://github.com/reactjs/reactjs.org/pull/1155